apps/pie-docs/src/_includes/head.njk

```diff
@@ -15,6 +15,7 @@
     <link rel="apple-touch-icon" sizes="180x180" href="{{ '/apple-touch-icon.png' | url }}">
     <link rel="icon" type="image/png" sizes="32x32" href="{{ '/favicon-32x32.png' | url }}">
     <link rel="icon" type="image/png" sizes="16x16" href="{{ '/favicon-16x16.png' | url }}">
+     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3">
```

Comment:
Ideally we would be able to get this from an npm package. Can we avoid loading directly from a CDN @LTurns ? We've done similar for other CSS files (such as fozzie cookie banner, I believe)